main.py

```diff
@@ -3,8 +3,22 @@
 
 def main():
     """Função principal que será rodada quando o script for passado para o interpretador."""
-    # COLOQUE SEU CÓDIGO AQUI
+    a = float(input("Digite o primeiro número\n"))
+    b = float(input("Digite o segundo número\n"))
+    c = float(input("Digite o terceiro número\n"))
+    v = [1,2,3,4]
+   
+    v[1] = a
+    v[2] = b
+    v[3] = c
+    if v[3] > v[2]:
+      v[2] = v[3]
 
+    if v[2] > v[1]:
+      v[1] = v[2]
+    
+    valor = v[1]
```

Comment:
Não entendi porque fazer isso?
Era para ter uma lista com 4 espaços vazios?
Se for isso, sugiro mudar para:
```suggestion
    v = list() # cria uma lista vazia
   
    v.append(a) # adiciona o a na lista
    v.append(b)
    v.append(c)
    if v[2] > v[1]: # como lista começa com o índice 0, o v[2] é o último elemento dessa lista com 3 números.
      v[1] = v[2]

    if v[1] > v[0]:
      v[0] = v[1]
    
    valor = v[0]
```

Reply:
@DnlSzin

Reply:
quando comecei a fazer tinha esquecido por um momento que o 0 conta como o primeiro valor da lista, então ao começar o vetor tinha atribuido valores de 1 a 3, para depois substituir, de fato dessa maneira é bem mais pratico, tendo em vista até que o append já atribui valor sem eu ter q colocar um valor aleatorio antes, irei concertar